src/main/java/ru/teti/springcourse/config/SpringConfig.java

```diff
@@ -18,7 +17,6 @@
 public class SpringConfig  implements WebMvcConfigurer {
     private final ApplicationContext applicationContext;
 
-    @Autowired
```

Comment:
@Autowired annotation is not mandatory here, injection will be performed anyway. But it's up to you, maybe you'll decide to use it here for better injection markup.